app/views/nfg_ui/_logo.html.haml

```diff
@@ -0,0 +1,12 @@
+- bonterra = false if local_assigns[:bonterra].nil?
```

Comment:
i'm fine with this, but are we setting this to false so that we can gate when we update different products or to preserve nfg logo preference? Just curious

Reply:
Set it to false since we are not promoting the Bonterra logo within the admin areas and only public for now. In the future, this will probably be changed to show only one bonterra logo across both OR two versions (public vs admin). I'm up for best practice to how it should be written in the code so long as the output in the same in the apps.

Reply:
This is fine then. good to go.